README.md

```diff
@@ -2,8 +2,7 @@ regenerator [![Build Status](https://travis-ci.org/facebook/regenerator.svg?bran
 ===
 
 This package implements a fully-functional source transformation that
-takes the proposed syntax for generators/`yield` from future versions of
-JS ([ECMAScript6 or ES6](http://wiki.ecmascript.org/doku.php?id=harmony:specification_drafts), experimentally implemented in Node.js v0.11) and
+takes the syntax for generators/`yield` from [ECMAScript 6 or ES6](http://www.ecma-international.org/ecma-262/6.0/) and [Asynchronous Iteration](https://github.com/tc39/proposal-async-iteration) proposal and
```

Comment:
It's now called ECMAScript 2015 or ES2015, if you don't mind changing that (and rebasing to fix the tests).